File: database/src/main/java/com/example/database/dtos/UserStatsGetDTO.java
package com.example.database.dtos;

import com.example.database.entities.Activity;
import com.example.database.entities.ActivityType;
import lombok.Builder;
import lombok.Value;
import lombok.extern.jackson.Jacksonized;

@Value
@Builder
@Jacksonized
public class UserStatsGetDTO {

    UserDTO userDTO;

    Integer avgCalories;

    ActivityType favActivityType;

    Activity lastActivity;

}

File: MyApplication/app/src/main/java/com/example/myapplication/dtos/UserStatsGetDTO.kt
package com.example.myapplication.dtos

import com.example.myapplication.model.Activity
import com.example.myapplication.model.ActivityType

data class UserStatsGetDTO(
    val userDTO: UserDTO? = null,
    val avgCalories: Int = 0,
    val favActivityType: ActivityType? = null,
    val lastActivity: Activity? = null
)